Formula/java-deobfuscator.rb

```diff
@@ -0,0 +1,19 @@
+class JavaDeobfuscator < Formula
+  desc "Tool aimed to deobfuscate most available obfuscators for Java"
+  homepage "https://github.com/java-deobfuscator/deobfuscator"
+  url "https://ci.samczsun.com/job/java-deobfuscator/job/Deobfuscator/lastSuccessfulBuild/artifact/target/deobfuscator-1.0.0.jar"
+  sha256 "27d3da75c5398d7eeb8e1790c0dcef99c47a506caece26ceef95646028f6db08"
+
+  bottle :unneeded
+
+  depends_on :java => "1.7+"
+
+  def install
+    libexec.install "deobfuscator-#{version}.jar"
+    bin.write_jar_script libexec/"deobfuscator-#{version}.jar", "java-deobfuscator"
+  end
+
+  test do
+    system "false"
```

Comment:
Hey there, this line is preventing the test bot from completing what it needs to.  Can you write/include a test for this formula?

Reply:
I can if you dont mind that `test` block will be really huge in comparison with `install` block :)

Reply:
Not at all, thank you very much!  😄

Reply:
@vinyldarkscratch Well, definitely not for now - during writing the 'testcase' I found that `java-deobfuscator` uses `sunmscapi.jar`, which is Windows dependent external lib.
I opened an issue, let's see what will happen.

Reply:
*!  Since all new formulas require tests, we’ll have to hold off before we’ll be able to get this merged, sadly.

Reply:
@vinyldarkscratch No problem, maintainer said he fixed this issue, I will check it soon.